website/_posts/2023-03-31-what-is-testnet.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "What is Testnet?"
+date: 2023-03-01 00:00:00 +0000
+tags: learn
+summary: What is Testnet?
+---
+
+## Mainnet vs Testnet?
+If you have ever wondered what Testnet is and how it is different from Mainnet, read this article to the end to find out!
+
+Mainnet and Testnet are two widely used terminologies in the Blockchain ecosystem; so it is important to understand the meaning and differences while working in the industry or following announcements of the Blockchain projects like Pactus.
+
+Here’s a quick explanation:
+Before the actual live version of a Blockchain network that is used for real transactions, functions and smart-contracts (which is referred as Mainnet), there is a test version of the Blockchain network for testing purposes (which is referred as Testnet). In Testnet, a safe environment is provided to help developers experiment and test their applications, and make sure that everything functions smoothly in the Mainnet version.
+
+## What is Testnet?
+A Testnet is a simulation of a Blockchain, in which developers can test all the functionalities of their Blockchain network without risking their assets and convenience of end-users. Using Testnet, any possible error, bug or malfunctions can be fixed before the Mainnet launch.
+Testnet also can be an experimental environment for new features, updates and upgrades. So several testnet networks of the same version or newer version of the Mainnet can be implemented on the roadmap.
+
+
+## What are use cases of testnet?
+**A secure environment to test development ideas and updates.** So 	many features and minor details in Pactus project have been 	successfully developed so far using the Testnet; such as:
+
+- extra details and information on Pactus-GUI;
+ 	
+- well-functioning 	Pactus Blockchain explorer to access transactions and block details;
+ 	
+- Upgrades 	that led to decreasing the size of the Pactus network
+
+**Providing 	a safe testing ground for resolve scalability and security problems, 	as well as decentralization issues.** Pactus 	network is also a secure and purely decentralized network. However 	in the last updates during the Testnet, 	developers are now 	able to decrease 	the database size by approximately 10% without compromising 	performance. They did it by utilizing block height instead of hash 	to store blocks within the database. (you can find out more here).
```

Comment:
```suggestion
**Providing a safe testing ground for resolve scalability and security problems, as well as decentralization issues.** Pactus network is also a secure and purely decentralized network. However in the last updates during the Testnet, developers are now able to decrease the database size by approximately 10% without compromising performance. They did it by utilizing block height instead of hash to store blocks within the database. (you can find out more [here](https://github.com/pactus-project/pactus/pull/416)).
```

fixing spacing issues and adding the link